src/main/java/com/github/onlinebookstore/controller/BookController.java

```diff
@@ -22,42 +23,50 @@
 @RestController
 @RequestMapping("/books")
 public class BookController {
+    private static final String HAS_ROLE_USER = "hasRole('ROLE_USER')";
+    private static final String HAS_ROLE_ADMIN = "hasRole('ROLE_USER')";
```

Comment:
It's okay. But not a conventional approach to extract value of @PreAuthorize to constant